learn.qmd

```diff
@@ -1,13 +1,73 @@
+
+### e-Epi training kit en Español
+
+El **e-Epi-training kit** es una estrategia para el aprendizaje en línea que permite un aprendizaje progresivo y flexible. Este kit incluye el curso virtual en ciencia de datos en salud pública y modelamiento de enfermedades infecciosas, el cual está dirigido a América Latina y el Caribe y cuenta con un enfoque de género. [Leer más detalles sobre la estrategia en línea](https://epiverse-trace.github.io/epi-training-kit/).
+
+### Tutoriales en Español
+
+| Tutorial                                           | Sitio                                                                                                                                                    | Repositorio                                                                                        | Agenda para instructoras/es                                                                                                   | Fase de desarrollo                                                                                                                                                   |
+| -------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Análisis de Brotes y Modelamiento en Salud Pública | [{{< fa regular window-maximize title="Website">}}](https://epiverse-trace.github.io/epimodelac/) | [{{< fa brands github title="GitHub Repository" >}}](https://github.com/epiverse-trace/epimodelac/) | [{{< fa solid person-chalkboard title="Instructor view">}}](https://epiverse-trace.github.io/epimodelac/instructor/index.html) | [![Static Badge](https://img.shields.io/badge/pre-alpha-991b1b)](https://carpentries.github.io/lesson-development-training/lesson-design.html#iterative-development) |
+
+### Tutorials in English
+
+| Tutorial                                                         | Site                                                                                                    | Repository                                                                                              | Instructor Schedule                                                                                                                 | Development stage                                                                                                                                                    |
```

Comment:
Looking at how this looks on the cellphone (Please, see attached screenshot), I'm interested in finding ways to increase the width of the name column. 

Screenshot:
![image](https://github.com/epiverse-trace/epiverse-trace.github.io/assets/21230390/bed09dde-1f12-4759-ad4f-3686bcb1beb4)

I think:
- one could be to change the column name of the last column from `Development stage` to `Stage`. Any strong opinion on this?
- other, to combine the name and site columns, linking them directly in the name, as previously tested by @chartgerink in this PR.

Also, if we mix both alternatives we can 

```suggestion
| Tutorial                                                         | Site                                                                                                    | Repository                                                                                              | Instructor Schedule                                                                                                                 | Stage                                                                                                                                                    |
```

Reply:
for the record, this is the outlook of the name and site columns mixed from a cellphone

![WhatsApp Image 2024-02-29 at 13 03 39_740f334f](https://github.com/epiverse-trace/epiverse-trace.github.io/assets/21230390/612d5a93-ac0f-4cf1-8d23-795d0365a5cb)

Reply:
https://quarto.org/docs/authoring/tables.html#column-widths

Reply:
@avallecam please provide final input for the tables. I have changed them back and forth several times already, so in order to prevent any further issues, I will await your specific instructions for the final revision as part of this PR.

Reply:
I agree with the final table outlook. I moved extra suggestions to separate issues to solve in other PRs #207 and #208